elsie/utils/cache.py

```diff
@@ -4,11 +4,10 @@
 
 
 class FsCache:
-    def __init__(self, cache_dir, version, inkscape_version):
-        start = ("elsie-{}/{}/".format(version, str(inkscape_version))).encode()
+    def __init__(self, cache_dir, version):
```

Comment:
Inkscape version should be there, otherwise cache may be corrupted

Reply:
It is, see slidedeck.py:128 (https://github.com/spirali/elsie/pull/70/files#diff-4b8678483ad0b238be36aff422cdc12f8b4d6083e61302eea465bbadc09b1970R108).

Reply:
Ah I see, the version already contains a the Inkscape version